Tests/SwiftFrameTests/ColorHexTests.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+import XCTest
+@testable import SwiftFrameCore
+
+class ColorHexTests: XCTestCase {
+
+    func testCorrectColor() throws {
+        let color = try NSColor(hexString: "#ff0000")
+        var red: CGFloat = CGFloat.greatestFiniteMagnitude
+        color.getRed(&red, green: nil, blue: nil, alpha: nil)
+        XCTAssertEqual(red, 1.00)
```

Comment:
That would still succeed if `ff0000` would be converted into 255, 255, 255. You could maybe compare the resulting color to one created from the floating point components 1, 0, 0. Then you don't have to read the components with that awkward pointer API.